docker/dev/centos/README.md

```diff
@@ -0,0 +1,46 @@
+# How to use this Dockerfile
+
+This image is intended for development of Orion Context Broker. This will create an environment on which Orion can compile and be run based on CentOS 6.
+
+If you are only interested in running Orion Context Broker in a container this image is not what you are looking for. 
+
+## Build the image
+
+You only need to do this once.
+
+    docker build -t orion-dev:centos6 .
+    
+## Run the container
+
+This assumes that there is another docker running with name $mongodb$. If you don't, type this:
+
+    docker run -d --name mongodb mongo
+
+The following line will run the container exposing port 1026, give it a name -in this case $orionDev$-, link it to the mongodb docker, and present a bash prompt. Keep in mind that /path/to/orion/source is the path to your Orion Context Broker code in the host, not the container. 
+
+    docker run -p 1026:1026 --name orionDev -v ~/src/fiware-orion:/root/src --link mongodb:mongodb -t -i orion-dev:centos6 /bin/bash
+
+## Compile Orion
+
+Before you compile you must disable proxyCoap (not supported in this docker image):
```

Comment:
We should find a definitive way of "removing" proxyCoap for building framework (make, etc.). It is becoming an obstacle :(

However, not in the scope of this PR... Maybe the next one?

NTC